### ISSUE TITLE
Touch doesn't work in Suggest widget

### ISSUE THREAD

**Issue by filip-hejsek** [microsoft/vscode#78984]
- VSCode Version: Code 1.36.1 (2213894ea0415ee8c85c5eea0d0ff81ecc191529, 2019-07-08T22:59:35.033Z)
- OS Version: Windows_NT x64 10.0.17763

Steps to Reproduce:

1. Open some file in VSCode
2. Press Ctrl+Space
3. Touch some completion item. Nothing happens

If you don't have a touch device, you can open dev tools (Ctrl+Shift+I) and click Toggle device toolbar to emulate touch.

The issue is in `SuggestWidget` - it disables `mouseSupport` and then only handles mouse events (but `mouseSupport` also handles touch events)
https://github.com/microsoft/vscode/blob/5ea298a88f8528bfdd0d08fab08dbe1267b0eb7a/src/vs/editor/contrib/suggest/suggestWidget.ts#L503-L519

This was already once fixed in list widget (#9338), but then broken again in suggest widget.

**Comment by davidreis97** [comment]
Hi there, first time contributor here, I'd like to have a go at this one, but I'm having trouble replicating the issue, as you can see in the gif. Maybe this bug doesn't exist anymore?

![ezgif-2-1de54cbc85e8](https://user-images.githubusercontent.com/15610011/65336051-85b45280-dbbd-11e9-961b-96069ae21a82.gif)

**Comment by joaomoreno** [comment]
@davidreis97 Maybe it's a Windows only issue.

**Comment by davidreis97** [comment]
@joaomoreno Thanks for the tip, after further testing I found the issue does exist on both MacOS and Windows, I just wasn't emulating touch events properly on the dev tools. Looks like toggling device toolbar on the dev tools, as suggested by to emulate a mobile viewport, as suggested by @filip-hejsek, is not enough to turn mouse clicks into touch events, I actually had to set "Touch" to "Force enabled" on the Sensors settings of the dev tools.

I'll start looking into how to fix this.
